alembic_postgresql_enum/operations/sync_enum_values.py

```diff
@@ -204,11 +204,11 @@ def is_column_type_import_needed(self) -> bool:
 @alembic.autogenerate.render.renderers.dispatch_for(SyncEnumValuesOp)
 def render_sync_enum_value_op(autogen_context: AutogenContext, op: SyncEnumValuesOp):
     if op.is_column_type_import_needed:
-        autogen_context.imports.add("from alembic_postgresql_enum import ColumnType")
-    autogen_context.imports.add("from alembic_postgresql_enum import TableReference")
+        autogen_context.imports.add("from alembic_postgresql_enum import ColumnType  # type: ignore[attr-defined]")
+    autogen_context.imports.add("from alembic_postgresql_enum import TableReference  # type: ignore[attr-defined]")
```

Comment:
`from alembic_postgresql_enum.get_enum_data.types import TableReference` seems to work fine for me rather than doing the ignore?